libs/datagrid/src/lib/cells/stacked-cell.tsx

```diff
@@ -27,3 +27,28 @@ export const StackedCell = ({
     </div>
   );
 };
+
+export const StackedCell = ({
+  primary,
+  secondary,
+}: {
+  primary: ReactNode;
+  secondary: ReactNode;
+}) => {
+  return (
+    <div className="leading-4">
+      <div
+        className="text-ellipsis whitespace-nowrap overflow-hidden text-sm "
```

Comment:
non-blocking: Whitespace character at the end of this string.

Also, rather than duplicating the component with some minor changes or adding a primaryIcon prop. IMO passing in a different element to handle the icon layout is preferable. Something like this

```
<StackedCell
  primary={
    <span className="whatever-you-need">
       {value}
        <span>
          <MarketProductPill productType={productType} />
        </span>
        <span className="ml-0.5">
          <MarketIcon data={data} />
        </span>
     </span>
  }
/>
```

Reply:
agree! updated